pylint/checkers/exceptions.py

```diff
@@ -73,10 +73,10 @@ def _is_raising(body: list) -> bool:
         "is raised (i.e. a `TypeError` will be raised).",
     ),
     "E0703": (
-        "Exception context set to something which is not an exception, nor None",
-        "bad-exception-context",
+        "Exception cause set to something which is not an exception, nor None",
+        "bad-exception-cause",
         'Used when using the syntax "raise ... from ...", '
-        "where the exception context is not an exception, "
+        "where the exception cause is not an exception, "
         "nor None.",
```

Comment:
This need to define an ``"old_names"`` so the message control still work with ``bad-exception-context``. You can search for an example in the codebase.